Pioneer-DDJ-200-scripts.js

```diff
@@ -13,18 +13,23 @@ DDJ200.scratch = function (channel, control, value, status, group) {
     engine.scratchTick(script.deckFromGroup(group), value - 64);
 };
 
-DDJ200.jog = function (channel, control, value, status, group) {
+DDJ200.jog1 = function (channel, control, value, status, group) {
     // For a control that centers on 0x40 (64):
     // Convert value down to +1/-1
     // Register the movement
     engine.setValue(group, 'jog', value - 64);
 };
 
+DDJ200.jog = DDJ200.jog1;
+
 DDJ200.touch = function (channel, control, value, status, group) {
     var deckNumber = script.deckFromGroup(group);
     if (value == 0) {
         // disable scratch
+        DDJ200.jog = function () {}; // disable nudging to not prevent alignment
```

Comment:
I see what you are trying to do here. I have noticed this problem before, and this is probably the only solution to it.

I like the idea of what you are trying to do. I think that enabling or disabling the nudging could however be done more neatly by setting a global boolean here, and checking this boolean in jog(), as opposed to trying to use a 'jog' and 'jog1'. What do you think?

Reply:
It was the smallest save value.